Userland/Libraries/LibWeb/HighResolutionTime/Performance.cpp

```diff
@@ -65,11 +65,20 @@ JS::GCPtr<NavigationTiming::PerformanceNavigation> Performance::navigation()
     return m_navigation;
 }
 
+// https://w3c.github.io/hr-time/#timeorigin-attribute
 double Performance::time_origin() const
 {
+    // FIXME: The timeOrigin attribute MUST return the number of milliseconds in the duration returned by get time origin timestamp for the relevant global object of this.
     return static_cast<double>(m_timer.origin_time().nanoseconds()) / 1e6;
 }
 
+// https://w3c.github.io/hr-time/#now-method
+double Performance::now() const
+{
+    // The now() method MUST return the number of milliseconds in the current high resolution time given this's relevant global object (a duration).
+    return current_high_resolution_time(this->realm().global_object());
```

Comment:
You should be able to use `HTML::relevant_global_object(*this)` here.

Reply:
Fixed 👍